File: backend/src/models/Quotes.ts
import mongoose, { Schema, Document } from 'mongoose';

export interface IQuotes extends Document {
    content: string
    deleted: boolean
}

const QuotesSchema: Schema = new Schema({
    content: {
        type: String,
        required: [true, '句子不能为空'],
        trim: true
    },
    deleted: {
        type: Boolean,
        default: false
    }
}, {
    timestamps: true
})

const Quotes = mongoose.model<IQuotes>('Quotes', QuotesSchema);
export default Quotes;

File: backend/java-backend/src/main/java/com/example/demo/model/mongo/Quotes.java
package com.example.demo.model.mongo;

import lombok.Data;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

@Data
@Document(collection = "quotes")
public class Quotes {

    @Id
    private String id;

    private String content;
    private Boolean deleted;
}
